wait-for-it.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: wait-for-it
+  version: 0_git200823
+  epoch: 0
+  description: Pure bash script to test and wait on the availability of a TCP host and port
+  target-architecture:
```

Comment:
I think @imjasonh [comment](https://github.com/wolfi-dev/os/pull/163#discussion_r1043825188) still needs addressing.  I.e. we need a runtime dependency on bash
```
  dependencies:
    runtime:
      - bash
```

Reply:
What is the difference between the runtime dependencies and packages? Trying to learn. 😀